src/index.css

```diff
@@ -1,5 +1,192 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;700&display=swap');
+
 body {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  background-color: #efefef;
+}
+
+#root {
+  margin: 0 auto;
+  min-width: 750px;
+  max-width: 1200px;
+  font-family: 'Montserrat', sans-serif;
+}
+
+nav {
+  background-color: #fff;
+  padding: 2.5rem 0;
+}
+
+nav div,
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav div a {
+  text-decoration: none;
+  margin-right: 2.5rem;
+  font-weight: 600;
+  color: #aaa;
+}
+
+/* stylelint-disable-next-line selector-type-no-unknown */
```

Comment:
- [ ] Kindly remove this stylelint disable line its best to fix the issue in the code.